application/features/sync.feature

```diff
@@ -26,3 +26,10 @@ Feature: Synchronizing a record
       And the user does not exist in the ID Broker
     When I learn the user does not exist in the ID Store and I tell the ID Broker
     Then the ID Broker response should return an error
+
+  Scenario: User info in ID Broker does not equal user info in ID Store
+    Given the user exists in the ID Store
+      And the user exists in the ID Broker
+      And the user info in the ID Broker does not equal the user info in the ID Store
+    When I send the user info in the ID Store to the ID Broker
```

Comment:
You actually don't need to close the pull request. You can simply push changes to the branch, and the pull request will be update accordingly.